wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/Cells/ConversationCreateOptionsCell.swift

```diff
@@ -89,17 +94,18 @@ final class ConversationCreateOptionsCell: RightIconDetailsCell {
     }
 }
 
+// MARK: - ConversationCreateOptionsCell Extension
 extension ConversationCreateOptionsCell: ConversationCreationValuesConfigurable {
     func configure(with values: ConversationCreationValues) {
-        let guests = values.allowGuests.localized.localizedUppercase
-        let services = values.allowServices.localized.localizedUppercase
-        let receipts = values.enableReceipts.localized.localizedUppercase
+        let guests = values.allowGuests.localized.capitalized
```

Comment:
Do we need use `capitalized`? In general we should not use `capitalized` , because it uppercases the first character in each word, and we don't need it. We still have many places in the app where we need to fix it.

Reply:
I'm well aware of how `.capitalized` works. I think in this case it's just fine. We want to get rid of `.Uppercase` anyways. 

<img width="345" alt="Screenshot 2023-03-08 at 17 29 17" src="https://user-images.githubusercontent.com/10944108/223771729-dfae22c8-c5a1-4311-8d3d-2571d8924782.png">

Reply:
Yes, in this case it is not a problem, because "on" or "off" is just one word. But in general we shouldn't capitalize because by design we don't have use cases where we need to capitalize every word. I think the usage in this particular case can make the code confusing.
What if just` values.allowGuests.localized` only?

Reply:
If we don't use capitalized, then we will have to modify the Localizable.strings files to make the appropriate changes and I think we already dismissed that idea.

Reply:
Yes, we don't want to change Localizable.strings files.

Reply:
Then I think it is appropriate to keep it as it is (with the applied changes I made). Also, I checked where we use .capitalized so far and I didn't find any wrong use cases.